src/pages/ResearcherProfile.js

```diff
@@ -386,22 +386,6 @@ export default function ResearcherProfile(props) {
                 }),
               ]),
 
-              div({ className: 'col-xs-12' }, [
```

Comment:
Deleted this input block for academic/business email because the ticket asked us to... but! The input form is *always* disabled, so I don't see a harm to leaving it there. Shoudl this change be reverted?

Reply:
Probably best to get a second opinion, but my impression based on [DUOS-1818](https://broadworkbench.atlassian.net/browse/DUOS-1818) is that the intention is to remove all usages of academic email in the ui and then get rid of it altogether in consent

Reply:
Probably best to get a second opinion, but my impression based on [DUOS-1818](https://broadworkbench.atlassian.net/browse/DUOS-1818) is that the intention is to remove all usages of academic email in the ui and then get rid of it altogether in consent